generators/client/templates/src/test/javascript/_karma.conf.js

```diff
@@ -15,7 +15,7 @@ if (isDebug()) {
 module.exports = function (config) {
     config.set({
         // base path, that will be used to resolve files and exclude
-        basePath: '<%= TEST_SRC_DIR %>'.replace(/[^/]+/g,'..'),
```

Comment:
Please remove commits to the client. It will make merging harder for me. If you want to do any changes for client do a PR to master. Ng2 branch PRs should change only client2